faq.rst

```diff
@@ -334,63 +253,37 @@ are removed.
 
 I want to keep a separate working copy per development branch, is it possible?
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
-
-There are two ways:
-
-1) Use the "`share extension`_" as described in the :ref:`multiple-clones`
-   section;
-2) Create several clones of your local repository;
-
-If you want to use the second way, you can do::
-
-   $ hg clone cpython py35
-   updating to branch default
-   3434 files updated, 0 files merged, 0 files removed, 0 files unresolved
+Create several clones of your local repository::
+
+   $ git clone cpython py35
+   Cloning into 'py35'...
+   remote: Counting objects: 629890, done.
+   remote: Total 629890 (delta 0), reused 0 (delta 0), pack-reused 629889
+   Receiving objects: 100% (629890/629890), 216.81 MiB | 3.58 MiB/s, done.
+   Resolving deltas: 100% (503219/503219), done.
+   Checking connectivity... done.
    $ cd py35
-   $ hg update 3.5
-   86 files updated, 0 files merged, 11 files removed, 0 files unresolved
-
-The current branch in a working copy is "sticky": if you pull in some new
-changes, ``hg update`` will update to the head of the *current branch*.
+   $ git checkout 3.5
+   Switched to branch '3.5'
+   Your branch is up-to-date with 'origin/3.5'.
 
-.. _share extension: https://www.mercurial-scm.org/wiki/ShareExtension
 
-
-.. _hg-paths:
+.. _git-paths:
 
 How do I link my local repository to a particular remote repository?
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
 Your local repository is linked by default to the remote repository it
-was *cloned* from.  If you created it from scratch, however, it is not linked
-to any remote repository.  In ``.hg/hgrc`` file for the local repository, add
-or modify the following section::
-
-  [paths]
-  default = ssh://hg@hg.python.org/devguide
+was *cloned* from. If you would like to link to a different remote, first add
+the remote::
 
-This example is for a local repository that mirrors the ``devguide`` repository
-on ``hg.python.org``. The same approach works for other remote repositories.
+    $ git remote add myremote <remote url>
 
-Anywhere that ``<remote repository>`` is used in the commands in this
-FAQ, ``hg`` will use the default remote repository if you omit the parameter.
-
-
-How do I create a shorthand alias for a remote repository?
-''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
-
-In your global ``.hgrc`` file add a section similar to the following::
-
-  [paths]
-  dg = ssh://hg@hg.python.org/devguide
-
-This example creates a ``dg`` alias for the ``devguide`` repository
-on ``hg.python.org``. This allows "dg" to be entered instead of the
-full URL for commands taking a repository argument (e.g. ``hg pull dg`` or
-``hg outgoing dg``).
+In ``.git/config`` file for the local repository, add
```

Comment:
Thanks for the feedback! Is what I just pushed what you were thinking? I will admit to fiddling with the git config file more than I should do...